assets/src/edit-story/components/fontPicker/pickerContainer.js

```diff
@@ -132,10 +146,41 @@ function FontPickerContainer({ value, onSelect, onClose }) {
   useFocusOut(ref, onClose, [onClose]);
 
   // Scroll to offset for current value
-  const currentOffset = fonts.findIndex(({ name }) => name === value);
+  const [currentOffset, setCurrentOffset] = useState(
+    fonts.findIndex(({ name }) => name === value)
+  );
 
-  // This is static for now, but with search and used fonts, this will change later
-  const matchingFonts = fonts;
+  const matchingFonts = useMemo(() => {
+    if (searchKeyword.trim() === '') {
+      return fonts;
+    }
+    const _matchingFonts = fonts.filter(({ name }) =>
+      name.toLowerCase().includes(searchKeyword.toLowerCase())
+    );
+    if (_matchingFonts.length === 0) {
+      setCurrentOffset(-1);
+    }
+    return _matchingFonts;
+  }, [fonts, searchKeyword]);
+
+  const handleSearchInputChanged = useCallback(({ nativeEvent }) => {
+    setSearchKeyword(nativeEvent.target.value);
+  }, []);
+
+  const handleKeyPress = useCallback(
+    ({ nativeEvent: { code } }) => {
+      if (code === 'Escape') {
+        onClose();
+      } else if (code === 'Enter') {
```

Comment:
Maybe <kbd>Space</kbd> too?

Reply:
Should I just close the previous PR since this has the changes of the other one?